### PR TITLE
feat: Add support for Linux ARM64

### PR DESCRIPTION
**Pull request by tucksaun**
While working on #352 I discovered that there's currently no support for Linux ARM64.
But this could be convenient to have it when running a Docker container on an Apple Silicon machine.

So here it is if you want it :) 

Spoiler: didn't really test it

### CODE REVIEW

**Review by ruudk — COMMENTED** [review]
Nice, Can we do the same for Darwin?

**Comment by tucksaun** [comment]
@ruudk Darwin ARM64 is already supported 😉

**Comment by ruudk** [comment]
Ah sorry I thought this was about the compiled version. But it's the PHAR one. I hope to get a compiled Darwin ARM64 version one day 😄

**Review by pyrech — APPROVED** [review]
Thanks :yellow_heart:

This would deserve a note in the changelog and the documentation IMO

File: tools/watcher/bin/watcher-linux-arm64
Comment (lyrixx) [inline tools/watcher/bin/watcher-linux-arm64:1]: Note for reviewers: we do not merge binary files committed by someone outside the jolicode organization. But we now pretty well Tugdual and we trust him

**Review by lyrixx — COMMENTED** [review]
Could you also edit the artifacts.yaml in .GitHub dir?

**Comment by tucksaun** [comment]
Changelog and `.github/workflows/artifacts.yml` updated.
I also added a note in the doc but not sure if this is what you imagined @pyrech so don't hesitate to rewrite or rephrase it :)

**Review by lyrixx — APPROVED** [review]
Thanks

**Pull request merged**